packages/@aws-cdk/aws-lambda-python/lib/Dockerfile.dependencies

```diff
@@ -1,22 +0,0 @@
-# The correct AWS SAM build image based on the runtime of the function will be
```

Comment:
We don't need a separate Dockerfile for installing dependencies since the install commands are customized depending on availability of dependency-files.